src/lib/app-switcher/app-switcher.component.html

```diff
@@ -0,0 +1,24 @@
+<button mat-icon-button [matMenuTriggerFor]="applicationMenu">
+  <mat-icon>apps</mat-icon>
+</button>
+<mat-menu #applicationMenu="matMenu" [overlapTrigger]="false" class="appswitcher-menu">
+  <div class="brand-container">
+    <mat-icon class="logo-icon" [svgIcon]="brandIcon"></mat-icon>
+    <div class="mat-body">{{ brandText }}</div>
```

Comment:
Should we allow `icon` and `text` to be configurable? 

If we are using it in products, we kinda want this to be the same across the board no?

Reply:
My only thought here is that if we make this non configurable. we would have to make this component dependent on a particular translation framework. Im ok with doing that since we currently all use NGX translate. but since thats now a non supported framework we have plans to move to transloco in the future....For the icon I had this configurable to switch out on theme switching and since each app currently has TD icons registered at the app level might be easier to pass that icon name and namespace instead of registering them within this component?

Reply:
The translation framework is gonna be a dependency regardless.. since we are gonna add a lot more components into the ui platform that are gonna require and its gonna be a pain to try and make applications pass the text to each one of them.. so i would suggest leverating `ngx-translate` for now and then we can change it to `transloco`

For the icon issue.. i understand that the app needs to register the svg's and also move them into the assets directory.. so i can go either way with this since the names need to be standard. So its either every app having to pass the names when the theme changes or the component itself doing it.

Ill leave the team to decide.

Reply:
IMO might as well add the pre-requisite for both icon name/namespace and translation string format. thoughts on `APP_SWITCHER.FOOTER` & `APP_SWITCHER.HEADER`?